plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -151,6 +144,11 @@ export async function start() {
     });
     theia.commands.registerCommand(VIEW, () => {
         viewPublicKey(sshKeyManager);
+        che.oAuth.getProviders().then(providers => {
+            providers.forEach(provider => {
+                console.log(provider);
```

Comment:
Is it some leftover or it's really needed?

Reply:
Removed the leftovers :)